content/posts/2017-vsc-plugin.md

```diff
@@ -0,0 +1,47 @@
++++
+draft = true
+date = "2017-03-08T10:36:40-05:00"
+title = "2017 vsc plugin"
+author = "Adam Bretz"
+categories = ["javascript", "vsc"]
++++
```

Comment:
You'll need to register for a [discourse](https://community.stardog.com/) account, and include your username as metadata here to enable commenting on the post.  The property name is `discourseUsername`.